conf/application.conf

```diff
@@ -165,6 +165,14 @@ microservice {
         base64KeystorePassword = ""
         certificate.pem = ""
       }
+      authorization {
+        # the key of the entry in the authorization header in the request going to NS&I -
+        # set to 'Authorization1` for local devleopment
+        header-key = "Authorization1"
+        user       = "user"
+        password   = "password"
+        encoding   = "UTF-8"
```

Comment:
I made it into a config variable just in case it isn't UTF-8 for production - I can code the stub to always accept UTF-8 but we don't have control over what NS&I will take